environment/plaintext/start.sh

```diff
@@ -36,6 +36,21 @@ else
   profile_grafana_command="--profile grafana"
 fi
 
+# defined 3 Connect variable and when profile is included/excluded
+profile_connect_nodes_command=""
+if [ -z "$ENABLE_CONNECT_NODES" ]
+then
+  log " Single connect node is being deployed"
+elif [ $(readlink -f "*.yml" | xargs -I {} -- sh -c "grep -hE  "connect.:" {} | wc -l") -gt 1 ] # Using grep and wc as simple grep with logical AND does not appear to work properly on yaml files. 
+then 
+  log " Found connect2 and connect3 in one or more of the following yaml files: $(readlink -f "*.yml" | xargs -I {} -- sh -c "grep -E  "connect.:" {} "). Multi node deployment will start shortly however it may still fail if services have missing configurations."
+  profile_connect_nodes_command="--profile connect_nodes"
+  export CONNECT_NODES_PROFILES="connect_nodes"
+else
+  log "🛑 Could not find connect2 and connect3 in any docker-compose*.yml override files. Update the yaml files to contain the connect2 && connect3 in $(readlink -f "*.yml") "
```

Comment:
Why is it required to override connect2 or connect3?

Reply:
In order to map every connectors volume properly and files located on the endpoint. The easiest example is JDBC connector where the driver is needed to be mapped:

`  connect:
    depends_on:
      - zookeeper
      - broker
      - schema-registry
      - mysql
    volumes:
        - ../../connect/connect-jdbc-mysql-sink/mysql-connector-java-5.1.45.jar:/usr/share/confluent-hub-components/confluentinc-kafka-connect-jdbc/lib/mysql-connector-java-5.1.45.jar
    environment:
      CONNECT_PLUGIN_PATH: /usr/share/confluent-hub-components/confluentinc-kafka-connect-jdbc`

Reply:
If connector gets deployed to the other nodes, it will fail with a missing driver. This is why the users need to ensure the override is done for the connectors on each node.

Reply:
Sure that makes totally sense